webapp/src/Form/Type/ContestType.php

```diff
@@ -94,6 +94,14 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
             ],
             'help' => 'Disable this to stop recording balloons. Usually you can just leave this enabled.',
         ]);
+        $builder->add('runtimeAsScoreTiebreaker', ChoiceType::class, [
+            'expanded' => true,
+            'choices' => [
+                'Yes' => true,
+                'No' => false,
+            ],
+            'help' => 'Enable this to show runtimes in milliseconds on scoreboard and use them as tiebreaker instead of penalty.',
```

Comment:
Help is updated. :+1: 

Regarding the runtime in seconds, here is a picture how it would look like.
![image](https://user-images.githubusercontent.com/12047669/219683795-1ae22e9f-813d-43be-9b4b-8f7c254d9f3a.png)
I think it looks more clean in milliseconds but if you prefer it in seconds I will commit the change.

Reply:
Just discussed this with @meisterT and he agreed that like this in seconds looks clearer.

My reasoning is that these are standard units; also, it makes it easier later to change the display format for submissions where the runtime is tens or hundreds of seconds and we might truncate some decimals.

Reply:
done :)